samples/sensor/max17262/README.rst

```diff
@@ -0,0 +1,41 @@
+.. _max17262:
+
+MAX17262 Fuel Gauge Sensor
+###################################
+
+Overview
+********
+
+This sample application periodically reads voltage, current and temperature data from
+the MAX17262 device that implements SENSOR_CHAN_GAUGE_VOLTAGE, SENSOR_CHAN_GAUGE_AVG_CURRENT,
+and SENSOR_CHAN_GAUGE_TEMP.
+
+Building and Running
+********************
+
+This sample application uses an MAX17262 sensor connected to a board via I2C.
+Connect the sensor pins according to the connection diagram given in the `max17262 datasheet`_.
+
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/sensor/max17262
+   :board: nrf52840dk_nrf52840
+   :goals: flash
+   :compact:
```

Comment:
These instructions won't compile for this board with the sample as-is, because there's no devicetree overlay which provides the appropriate node. Please fix this. I think it would be best to add a single `app.overlay` file that uses `arduino_i2c` as the bus node and updating the documentation to say that any board with Arduino I2C pins will work by default.

Reply:
I added `app.overlay` file, is it okay now?

Reply:
> I added `app.overlay` file, it should be fine now since build is passing.

I think that's OK for now. We have some plans to fix up the way samples get access to devices which are described in https://github.com/zephyrproject-rtos/zephyr/issues/32139. I will update this sample to use the new style when I am able to address this issue.